### PR TITLE
Add smug mug to imports

### PR DESCRIPTION
**Pull request by olsona**

### CODE REVIEW

File: extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugPhotosImporter.java [hunk 1]
```diff
@@ -106,15 +106,15 @@ void importSingleAlbum(UUID jobId, PhotoAlbum inputAlbum, SmugMugInterface smugM
...
 
   @VisibleForTesting
   void importSinglePhoto(UUID jobId, PhotoModel inputPhoto, SmugMugInterface smugMugInterface)
       throws IOException {
     // Find album to upload photo to
-    String newAlbumUri =
-        jobStore.findData(jobId, createCacheKey(), TempPhotosData.class)
-            .lookupNewAlbumId(inputPhoto.getAlbumId());
+    TempPhotosData tempPhotosData = jobStore.findData(jobId, createCacheKey(), TempPhotosData.class);
+    String newAlbumUri = tempPhotosData.lookupNewAlbumId(inputPhoto.getAlbumId());
```
Comment (holachuy) [inline extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugPhotosImporter.java]: Let's log if it's empty

Comment (olsona) [reply extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugPhotosImporter.java]: Added logging to RetryingCallable, should log every error that it catches.

File: extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugPhotosImporter.java [hunk 2]
```diff
@@ -106,20 +106,21 @@ void importSingleAlbum(UUID jobId, PhotoAlbum inputAlbum, SmugMugInterface smugM
...
 
   @VisibleForTesting
   void importSinglePhoto(UUID jobId, PhotoModel inputPhoto, SmugMugInterface smugMugInterface)
       throws IOException {
     // Find album to upload photo to
-    String newAlbumUri =
-        jobStore.findData(jobId, createCacheKey(), TempPhotosData.class)
-            .lookupNewAlbumId(inputPhoto.getAlbumId());
+    TempPhotosData tempPhotosData = jobStore.findData(jobId, createCacheKey(), TempPhotosData.class);
+    checkState(tempPhotosData != null,
```
Comment (rtannenbaum) [inline extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugPhotosImporter.java]: nit: can use checkNotNull(tempPhotosData)

File: portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/retry/RetryingCallable.java [hunk 3]
```diff
@@ -60,6 +64,7 @@ public T call() throws RetryException {
         return callable.call();
       } catch (Exception e) {
         mostRecentException = e;
+        LOGGER.error("Caught exception", e);
```
Comment (rtannenbaum) [inline portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/retry/RetryingCallable.java]: do you want to log attempt # too?

**Pull request merged**